src/main/sensors/gyro.c

```diff
@@ -294,7 +282,15 @@ bool gyroInit(void)
 {
     memset(&gyro, 0, sizeof(gyro));
 #if defined(USE_GYRO_MPU6050) || defined(USE_GYRO_MPU3050) || defined(USE_GYRO_MPU6500) || defined(USE_GYRO_SPI_MPU6500) || defined(USE_GYRO_SPI_MPU6000) || defined(USE_ACC_MPU6050) || defined(USE_GYRO_SPI_MPU9250) || defined(USE_GYRO_SPI_ICM20601) || defined(USE_GYRO_SPI_ICM20689)
-    gyroDev0.mpuIntExtiConfig = selectMPUIntExtiConfig();
+
+#if defined(MPU_INT_EXTI)
+    gyroDev0.mpuIntExtiConfig.tag =  IO_TAG(MPU_INT_EXTI);
+#elif defined(USE_HARDWARE_REVISION_DETECTION)
+    gyroDev0.mpuIntExtiConfig.tag =  selectMPUIntExtiConfigByHardwareRevision();
```

Comment:
It may be better to assign whole mpuIntExtiConfig, not just tag. But that will cause problems with structure definition.

Dropping `mpuIntExtiConfig` and just using `mpuIntTag` may be better